pkg/config/service.go

```diff
@@ -39,6 +41,7 @@ func GetServiceConfig() *ServiceConfig {
 		S3Prefix:           os.Getenv(s3PrefixEnvVar),
 		ClusterName:        os.Getenv(clusterNameEnvVar),
 		UsePathStyle:       os.Getenv(usePathStyleEnvVar) == "false",
+		EnableEncryption:   os.Getenv(enableEncryptionEnvVar) == "true",
```

Comment:
how does this default to encrypt? if it's not set this returns false and skips encryption

Reply:
i would restructure to have it enabled option and change to flag to be negated, i.e `DISABLE_ENCRYPTION`

Reply:
doing it right away